build-scripts/build-csdk.js

```diff
@@ -48,7 +49,8 @@ function findBins( iotivityPath ) {
 	return thePath;
 }
 
-var binariesSource, installWinHeaders, architecture, tinycborPath;
+var binariesSource, installWinHeaders, tinycborPath,
+	userAgentParts, platform, localArch, targetArch, sysVersion;
```

Comment:
I found targetArch(itecture) clearer since it matches the parameter for Scons. Are you're sure about this?